src/rust/engine/boxfuture/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock # See http://doc.crates.io/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries
```

Comment:
Apparently I gave you a bad example to crib - comments in .gitignore files must be at the start of the line. So this should be:
```
# See http://doc.crates.io/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries
Cargo.lock
```

sorry about that!